.github/workflows/tests.yaml

```diff
@@ -195,10 +195,58 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           files: ./coverage.xml
 
+  tests-qchem:
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: ["3.10"]
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: pip
+          cache-dependency-path: setup.py
+
+      - name: Set up conda
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          activate-environment: quacc-env
+
+      - name: Install pip packages
+        run: |
+          pip install .[strict,dev,qchem]
+          pip install chardet
+          pip install pytest-cov
+
+      - name: Install conda packages
+        run: |
+          conda install -c conda-forge openbabel
+
+      - name: Run tests with pytest
+        run: pytest tests/calculators/qchem tests/custodian_tests/test_qchem_custodian.py --cov=quacc --cov-report=xml
```

Comment:
Here I specify which tests to run so the qchem github actions runner isn't wasting time doing the whle unit test suite.